monitoring/caddy/caddy/Caddyfile

```diff
@@ -67,17 +86,17 @@ https://pushgateway.monitoring.gosh.sh {
 }
 
 https://next-app.monitoring.gosh.sh {
-    reverse_proxy http://gooosh.matusevich.pro {
-        header_up Host gooosh.matusevich.pro
+    reverse_proxy http://gosh.matusevich.pro {
```

Comment:
It should be passed in env variables to make it generic enough for tests

Reply:
https://app.asana.com/0/1202815812933539/1203788451585095/f

Reply:
This is not used anymore, will be removed in that task.